src/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
```

Comment:
you're using react 18, but not using its full potentials setting the project like this

.. here is a video solving this in the 1st minute https://youtu.be/N0DhCV_-Qbg

Reply:
What's wrong with this file?
I don't need the first line you mean `import React from 'react';`?

Reply:
watch the video I linked
there is another way to set a react project on version 18
you're setting the project the old way